src/types/plugin-inputs.ts

```diff
@@ -11,28 +11,43 @@ export interface PluginInputs<T extends SupportedEventsU = SupportedEventsU, TU
   ref: string;
 }
 
+export const approvalsRequiredSchema = T.Object(
+  {
+    /**
+     * The amount of validations needed to consider a pull-request by a collaborator to be deemed eligible for merge
+     */
+    collaborator: T.Number({ default: 1, minimum: 1 }),
```

Comment:
Defaults should be clearly documented in the readme so that there are no nasty surprises for future partners!

Reply:
Added comments within the readme and the code comments at https://github.com/ubiquibot/automated-merging/pull/1/commits/7677885c7014f6988d401051c26da93ac8e89744
Looking into knip.

Reply:
@0x4007 Knip is clean, it's just that the command doesn't get deleted once every issue is fixed. Latest Knip run: https://github.com/ubiquibot/automated-merging/actions/runs/9950037229/job/27487255118?pr=1